sample/src/main/java/io/ona/kujaku/sample/activities/CustomMarkerLowLevelAddPoint.java

```diff
@@ -0,0 +1,43 @@
+package io.ona.kujaku.sample.activities;
+
+import android.os.Bundle;
+import android.util.Log;
+import android.view.View;
+import android.widget.Button;
+
+import org.json.JSONObject;
+
+import io.ona.kujaku.sample.R;
+
+/**
+ * Created by Ephraim Kigamba - ekigamba@ona.io on 06/11/2018
+ */
+
+public class CustomMarkerLowLevelAddPoint extends LowLevelManualAddPointMapView {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        Button button = findViewById(R.id.btn_lowLevelManualAddPointMapView_doneBtn);
```

Comment:
This button is not visible for testing.

Reply:
The size has not been increased

Reply:
The button is still not visible.

Reply:
The visibility property is VISIBLE, unless I am wrong

Reply:
Fixed